ehrapy/tools/feature_ranking/_rank_features_groups.py

```diff
@@ -255,6 +257,7 @@ def rank_features_groups(
     correction_method: _method_options._correction_method = "benjamini-hochberg",
     tie_correct: bool = False,
     layer: Optional[str] = None,
+    rank_obs_columns: Optional[Union[list[str], str]] = None,
```

Comment:
1. Can we pls import `from __future__ import __annotations__`?
2. We can probably use a different type then `list`. `Iterable` or `Sequence`?

Reply:
1. Yes see no reason against that
2. Agree, had this in the first place - then ruff then yielded an error: `move_to_x` expecting list[str] | str, but incoming `Iterable[str]`. Suppose can just convert Iterable to list in `move_to_x`